prisma/schema.prisma

```diff
@@ -58,6 +58,8 @@ model Cohort {
   deliveryLogs DeliveryLog[]
   name         String?       @unique
   event        Event[]
+  createdAt    DateTime      @default(now())
+  updatedAt    DateTime?     @default(now())
```

Comment:
`updatedAt DateTime? @updatedAt`

No need to add the `@default` if you do this

Reply:
updated